apps/institution-service/cypress/e2e/aggregatorIntegrations.cy.ts

```diff
@@ -363,43 +418,54 @@ describe("POST /aggregatorIntegrations (Create)", () => {
       ) => {
         expect(response.status).to.eq(400);
 
-        expect(response.body.error).to.eq("Database Error");
+        expect(response.body.error).to.eq(
+          "Invalid reference in the field: providers_institution_id_fkey.",
```

Comment:
"provider" is left over from a database index that was created before the name change. I think when I fix the camelCase fields I will wipe everything and rebuild the schema from scratch and this will be renamed to "aggregator"